static/src/utils/store.ts

```diff
@@ -0,0 +1,31 @@
+import { reactive } from "vue";
+import { OpenAPI } from "@/api";
+import type { VueCookies } from "vue-cookies";
+
+export const store = reactive({
+  _jwt: undefined as string | undefined,
+
+  $cookies: undefined as VueCookies | undefined,
+
+  isLoggedIn: false,
+
+  initialize(cookies: VueCookies) {
+    this.$cookies = cookies;
+    this.jwt = cookies.get("jwt");
+  },
+
+  get jwt() {
+    return this._jwt;
+  },
+
+  set jwt(newJWT) {
+    this._jwt = newJWT;
+    this.isLoggedIn = !!newJWT;
+    OpenAPI.TOKEN = newJWT;
+    if (newJWT) {
+      this.$cookies?.set("jwt", newJWT);
+    } else {
+      this.$cookies?.remove("jwt");
+    }
+  },
```

Comment:
So if I call the `set jwt` without any parameter, that deletes the existing jwt, right?

Reply:
(What I actually wanted to say: is it common to do it that way or is it maybe better to define a `delete_jwt` method()?)

Reply:
Actually this is already how it's done :sweat_smile: